highway_env/road/road.py

```diff
@@ -262,10 +263,12 @@ def close_vehicles_to(self, vehicle: 'kinematics.Vehicle', distance: float, coun
                     if np.linalg.norm(v.position - vehicle.position) < distance
                     and v is not vehicle
                     and (see_behind or -2 * vehicle.LENGTH < vehicle.lane_distance_to(v))]
-        if sort:
-            vehicles = sorted(vehicles, key=lambda v: abs(vehicle.lane_distance_to(v)))
+
+        vehicles = sorted(vehicles, key=lambda v: abs(vehicle.lane_distance_to(v)))
         if count:
             vehicles = vehicles[:count]
+        if not sort:
```

Comment:
If sort is false, the vehicles will be sorted and then shuffled, which adds useless computation. It seems to me that the previous implementation was fine, wasn't it? (and unsorted does not necessarily mean randomly shuffled)

Reply:
The problem with the previous implementation was that doing:
```
if count:
    vehicles = vehicles[:count]
```
without sorting the vehicles before, resulted in taking N vehicles that wre not necessarily the N closest. And that was the real problem.
To resolve this the only way to do is to sort the vehicles before, and then doing `vehicles = vehicles[:count]`

Reply:
Aaaaah, I see. Nice catch!
Current implementation is fine, then.